src/Packetery/Module/Api/Internal/OrderController.php

```diff
@@ -187,9 +187,9 @@ public function saveModal( WP_REST_Request $request ) {
 				sprintf( '[data-packetery-order-id="%d"][data-packetery-order-grid-cell-weight]', $orderId ) => $this->gridExtender->getWeightCellContent( $order ),
 			],
 			Form::FIELD_WEIGHT        => $order->getFinalWeight(),
-			Form::FIELD_LENGTH        => $order->getLength(),
-			Form::FIELD_WIDTH         => $order->getWidth(),
-			Form::FIELD_HEIGHT        => $order->getHeight(),
+			Form::FIELD_LENGTH        => $this->helper::simplifyFloat( $order->getLength(), 1 ),
```

Comment:
K čemu je zaokrouhlení tady? Neměly by být uložené správné hodnoty?

Reply:
Odstraneno.

Reply:
Kdyz nad tim tam premyslim, tak asi by to bylo vhodnejsi udelat to tady, protoze ted to stejne delame na dvou mistech. V Metaboxu a v Modalu, kdy pred ulozenim ten float zredukujeme na 1 desetinne misto. Co si o tom myslis?